db/src/db.rs

```diff
@@ -69,12 +69,16 @@ pub fn new_connection<T>(uri: T) -> rusqlite::Result<rusqlite::Connection> where
         _ => rusqlite::Connection::open(uri)?,
     };
 
+    /// temp_store=2 is present to force SQLite to store temp files in memory.
+    /// Some of the platforms we support do not have a tmp partition (e.g. Android)
+    /// necessary to store temp files on disk.
```

Comment:
Please also refer to issue #505, and mention that it has pointers to Firefox's configuration.

We rely here on consumers being built using an appropriate set of flags; it's probably worth mentioning that.